lib/utils.js

```diff
@@ -647,8 +647,6 @@ exports.stackTraceFilter = function() {
   function isMochaInternal(line) {
     return (
       ~line.indexOf('node_modules' + slash + 'mocha' + slash) ||
-      ~line.indexOf('node_modules' + slash + 'mocha.js') ||
-      ~line.indexOf('bower_components' + slash + 'mocha.js') ||
       ~line.indexOf(slash + 'mocha.js')
```

Comment:
@outsideris L650~651 were removed because L652 already has them covered. The windows issues were caused by not using `path.seq`, which was fixed by assigning `slash` to `path.sep` instead of `/`.

Reply:
@plroebuck 	please add a comment about what you changed when editing comments/descriptions

Reply:
@boneskull The diff can be found by clicking the `edited by ...` link. He fixed a typo in my comment, gracias 😆